Makefile

```diff
@@ -0,0 +1,10 @@
+###> deploy to app engine ###
```

Comment:
I was following the symfony model of using a Makefile. But I can change this. It would then look like `composer run-script deploy`  and `composer run-script server` or something.